Lib/test/test_logging.py

```diff
@@ -547,7 +547,7 @@ def test_name(self):
     def test_builtin_handlers(self):
         # We can't actually *use* too many handlers in the tests,
         # but we can try instantiating them with various options
-        if sys.platform in ('linux', 'darwin'):
+        if MACOS or sys.platform == 'linux':
```

Comment:
The tests are failing because MACOS is not imported (I assume you mean support.MACOS).